config/deploy/migrate.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+server 'preservation-catalog-migrate.stanford.edu', user: 'pres', roles: %w[app db web resque queue_populator]
```

Comment:
this pres cat instance will have its own db and redis instances, right?

Reply:
(i.e. it won't share with prod or stage?)

Reply:
That's correct, the vm has its own db and redis instance already configured.